docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile

```diff
@@ -1,9 +1,6 @@
 ARG PYTORCH_VERSION=latest
```

Comment:
do we still need this file if we are adding rocdecode dependency. This was strictly for graphics drivers

Reply:
Deleted this file